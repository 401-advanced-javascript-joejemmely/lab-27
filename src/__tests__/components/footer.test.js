import React from 'react';
import rendeder from 'react-test-renderer';
import Footer from '../../components/footer/footer';

describe('Footer', () => {
  it('renders', () => {
    const mountedSimple = shallow(<Footer />);
    expect(mountedSimple.find('footer')).toBeTruthy();
  });

  it('rendering follows the snapshot', () => {
    const snapshot = rendeder.create(<Footer />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
