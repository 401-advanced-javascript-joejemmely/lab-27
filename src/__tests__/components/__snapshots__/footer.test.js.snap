// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer rendering follows the snapshot 1`] = `
<footer
  className="css-lte0pa"
>
  Â© 2019 Code Fellows
</footer>
`;
